.gitattributes

```diff
@@ -0,0 +1,14 @@
+# Path-based git attributes
```

Comment:
I don't think this file will be necessary with a proper `.gitignore` configuration.

Reply:
It's for download with compose

Reply:
Why? Sorry, I'm not sure to understand the purpose of your change. 😕

Reply:
When you download the project via a method other than the clone, it removes the elements defined here

Reply:
Got it! That's a really nice feature that I didn't know. You can probably simplify a lot the deployment script thanks to that. 😄